server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -197,8 +218,7 @@ private DivTag renderApplicationBlock(
         .withClasses(ReferenceClasses.ADMIN_APPLICATION_BLOCK_CARD, "w-full", "shadow-lg", "mb-4");
   }
 
-  private DivTag renderAnswer(
-      long programId, AnswerData answerData, boolean hasEligibilityEnabled) {
+  private DivTag renderAnswer(long programId, AnswerData answerData, boolean doesEligibilityApply) {
```

Comment:
**suggestion (non-blocking):** Consider documenting the doesEligibilityApply param since it's not immediately obvious what it means, or renaming it to something like showEligibilityText since that's a bit more obvious and the method doesn't need to know why to or to not show the line.

Reply:
Makes sense - updated to showEligibilityText.